### ISSUE TITLE
Missing WPA3 support in Game Mode

### ISSUE THREAD

**Issue by AkazaRenn** [ublue-os/bazzite#1100]
### Describe the bug

Unable to connect to WPA3 encrypted Wi-Fi in Game Mode, it is supported by Steam OS. 

In Bazzite, by selecting the SSID, no password input box is available if that's encrypted by WPA3, thus I cannot input the password.

### What did you expect to happen?

Same as WPA2, an input box for password should be available to provide the password so I can connect to that Wi-Fi.

### Hardware

Steam Deck LCD 512GB

**Comment by KyleGospo** [comment]
Unable to reproduce

**Comment by AkazaRenn** [comment]
@KyleGospo I cannot call out the keyboard on desktop mode either... Maybe it's because I ran the installer from Ventoy?

**Comment by klardotsh** [comment]
I was able to repro OP. Had to BOTH plug in a physical keyboard AND connect to my 2.4GHz/WPA2 fallback network to get through system setup; could not connect to my 5GHz/WPA3 primary network until I booted into Steam, took Steam client updates from the Steam settings menu, and let it soft-reboot (I presume this is a compositor+client restart). On that second "firstboot" menu (weird that I got dumped back to "firstboot" again to begin with, but I digress), my 5GHz/WPA3 network was connectable, AND the on-screen keyboard showed up no problem.
